src/Entity/ItemUpdate.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Table
+ * @ORM\Entity
+ * @ORM\HasLifecycleCallbacks
+ */
+class ItemUpdate
+{
+    /**
+     * @var UuidInterface
+     *
+     * @ORM\Id
+     * @ORM\Column(type="uuid", unique=true)
+     */
+    protected $id;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(type="text", nullable=false)
+     */
+    protected $secret;
+
+    /**
+     * @var Item
+     *
+     * @ORM\OneToOne(targetEntity="App\Entity\Item", inversedBy="update")
+     */
+    protected $item;
+
+    /**
+     * @var User
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\User")
+     */
+    protected $updatedBy;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    protected $lastUpdated;
+
+    public function __construct(Item $item, User $user)
+    {
+        $this->id = Uuid::uuid4();
+        $this->updatedBy = $user;
+
```

Comment:
extra spaces

Reply:
To separate internal entity logic and contructor parameters maping.